### PR TITLE
fix: align Safe Account, Safe{Wallet} and Safe Token branding

### LINKED ISSUE #1876: Rename "Safe" to "Safe{Wallet}"
## What is the feature about

We need to replace the Safe logo with Safe{Wallet}, as well as rename all instances of "Safe" to "Safe{Wallet}" as well

## The list of requirements

- Replace header image with that of Safe{Wallet}
- Find all instances of "safe"/"Safe" and adjust accordingly

Note: we cannot do a global find and replace as there are instances of "Safe" that not need to be replaced, e.g. "Safe Apps".

## Links

- [Discussion thread](https://5afe.slack.com/archives/C03FNGQ1LTU/p1678963381883159)
- [Safe Media Kit](https://safe-global.notion.site/Safe-Media-Kit-35ce7ffc829c4bedbbf828464a1b7c00)

### PR DESCRIPTION
**Pull request by iamacook**
## What it solves

Partially resolves #1876

## How this PR fixes it

All of the following instances have been updated according to our [brand map](https://docs.google.com/presentation/d/1c7IHqREWFvKtc5ngV1uZBi9eDp2ocHld0fWLBkTyQPs).

"Safe" (project) -> "Safe" (italicised)
"Safe" (contract) -> "Safe Account"
"Safe" (interface) -> "Safe{Wallet}"
Token -> "SAFE" or "Safe Token"

## How to test it

Observe the correct mapping in the changed files.

### CODE REVIEW

**Comment by iamacook** [comment]
This has just been [approved by Jorge](https://5afe.slack.com/archives/C03FNGQ1LTU/p1681906916291089?thread_ts=1681901771.664039&cid=C03FNGQ1LTU).

File: src/components/common/CookieBanner/index.tsx [hunk 1]
```diff
@@ -113,7 +113,8 @@ const CookieBannerPopup = ({ warningKey }: { warningKey?: CookieType }): ReactEl
                   />
                   <br />
                   <Typography variant="body2" color="background.paper">
-                    Help us make the app better. We never track your Safe or wallet addresses, or any transaction data.
+                    Help us make the app better. We never track your Safe Account or wallet addresses, or any
```
Comment (katspaugh) [inline src/components/common/CookieBanner/index.tsx:116]:
```suggestion
                    Help us make the app better. We never track your Safe Account address or wallet addresses, or any
```

**Review by katspaugh — APPROVED** [review]
Looks good!
"Safe Account" is quite brilliant.

**Review by schmanu — APPROVED** [review]
💯

**Comment by liliya-soroka** [comment]
Verified

- We remember about "Governance app"  and will update it too
- Description of the WC and MMi should be updated by safe- apps team on prod

**Pull request merged**